packages/inline-edit/src/interfaces.ts

```diff
@@ -18,6 +18,8 @@ export type InlineEditTheme = ITheme<IInlineEditTheme>;
 export interface IFieldProps<T> {
   value: T;
   onChange: (e: T) => void;
+  invalid?: boolean;
+  errorMessage?: string;
```

Comment:
`errorMessage` can be like string or array string and I think it's better to call `error`

Reply:
Should I change the value of this field? I mean, 
`error?: string | string[];`